app/core/tests/test_api.py

```diff
@@ -381,3 +382,28 @@ def test_create_soc_major(auth_client):
     res = auth_client.post(SOC_MAJOR_URL, payload)
     assert res.status_code == status.HTTP_201_CREATED
     assert res.data["title"] == payload["title"]
+
+
+def test_project_sdg_xref(auth_client, project, sdg, sdg1):
+    def get_object(data, target_uuid):
+        for obj in data:
+            if str(obj["uuid"]) == str(target_uuid):
+                return obj
+        return None
+
+    project.sdgs.add(sdg)
+    project.sdgs.add(sdg1)
+    proj_res = auth_client.get(PROJECT_URL)
+    test_proj = get_object(proj_res.data, project.uuid)
+    assert test_proj is not None
+    assert len(test_proj["sdgs"]) == 2
+    assert test_proj["sdgs"][1] != sdg.name
+    assert test_proj["sdgs"][0] == sdg.name
+    assert test_proj["sdgs"][1] == sdg1.name
+
+    sdg.projects.add(project)
```

Comment:
Yes, they should be related through the add statements from above. I removed that line now and confirmed that the GET request returns sdgs and the associated projects list.

Reply:
FYI: made the update and it complained about ```objs``` as a name (something about clarifying it). My terminal output is gone now, so I don't know the exact error. Instead of ```objs```, I renamed to ```objects``` and updated the changes I made in the docs as well.